nixos/tests/3proxy.nix

```diff
@@ -134,6 +134,8 @@ import ./make-test-python.nix ({ pkgs, ...} : {
   };
 
   testScript = ''
```

Comment:
Any reason not to start the machines simultaneously? It will speed up the test.

```suggestion
  testScript = ''
    start_all()

```

Reply:
Fixed, thanks!